js/toggle.js

```diff
@@ -6,12 +6,13 @@ jQuery(document).ready(function() {
 	else
 		expand_default(selected);
 	jQuery(window).on('load',function () {
-		if(item_type == "none")
+		if(item_type == "none") {
 			item_hidden();
-		else if(selected != "1" && item_type != "none")
+		} else if(selected != "1" && item_type != "none") {

```

Comment:
if( ( item_type == "none" ) || ( selected != "1" && item_type != "none" ) {
			item_hidden();

can we combine these 2 conditions?

Reply:
This is updated @sushmak02